component/service/src/main/java/org/exoplatform/social/rest/api/ProfileLabelRest.java

```diff
@@ -0,0 +1,274 @@
+      return Response.ok().build();
+    } catch (ObjectAlreadyExistsException ex) {
+      LOG.warn("Cannot add a profile label with type {}, object Id {} and language {} , it is already created",
+               profileLabel.getObjectType(),
+               profileLabel.getObjectId(),
+               profileLabel.getLanguage(),
+               ex);
+      return Response.status(Status.CONFLICT).build();
+    } catch (Exception ex) {
+      LOG.warn("Failed to create a new profile label with type {}, object Id {} and language {}",
+               profileLabel.getObjectType(),
+               profileLabel.getObjectId(),
+               profileLabel.getLanguage(),
+               ex);
+      return Response.status(HTTPStatus.INTERNAL_ERROR).build();
+    }
+  }
+
+  @POST
+  @Path("labels")
+  @RolesAllowed("administrators")
+  @Operation(summary = "Creates a list of new Labels", method = "POST", description = "Creates a list of Labels.")
+  @ApiResponses(value = { @ApiResponse(responseCode = "200", description = "Request fulfilled"),
+      @ApiResponse(responseCode = "500", description = "Internal server error"),
+      @ApiResponse(responseCode = "400", description = "Invalid query input"),
+      @ApiResponse(responseCode = "401", description = "Unauthorized operation") })
+  public Response createLabels(@Context
+  UriInfo uriInfo,
+                               @Parameter(description = "Object type", required = true)
+                               @RequestBody(description = "Profile label object to be created", required = true)
+                               List<ProfileLabel> profileLabels) {
+    if (profileLabels == null || profileLabels.isEmpty()) {
+      return Response.status(Status.BAD_REQUEST).entity("Labels list is empty").build();
+    }
+    try {
+      for (ProfileLabel label : profileLabels){
+        profileLabelService.createLabel(label);
+      }
+      return Response.ok().build();
+    } catch (Exception ex) {
+      LOG.warn("Failed to create  the list of profile labels f", ex);
+      return Response.status(HTTPStatus.INTERNAL_ERROR).build();
+    }
+  }
+
+  @PUT
+  @RolesAllowed("administrators")
+  @Operation(summary = "Update a Profile label", method = "PUT", description = "update a Profile label.")
+  @ApiResponses(value = { @ApiResponse(responseCode = "200", description = "Request fulfilled"),
+      @ApiResponse(responseCode = "500", description = "Internal server error"),
+      @ApiResponse(responseCode = "400", description = "Invalid query input") })
+  public Response updateLabel(@Context
+  UriInfo uriInfo, @RequestBody(description = "Profile label object to be updated", required = true)
+  ProfileLabel profileLabel) {
+    if (profileLabel == null) {
+      return Response.status(Status.BAD_REQUEST).entity("Profile label object is null").build();
+    }
+    try {
+      profileLabelService.updateLabel(profileLabel);
+      return Response.ok().build();
```

Comment:
To apply on all empty responses please.

```suggestion
      return Response.noContent().build();
```